### ISSUE TITLE
Incorrect renaming for these files

### ISSUE THREAD

**Issue by djkrimson** [tvrenamer/tvrenamer#93]
Incorrect renaming for these files on yosemite

Strike.Back.S01E01.Mini.720p.HDTV.DD5.1.x264.mkv
Strike.Back.S01E02.Mini.720p.HDTV.DD5.1.x264-ITSat.mkv
Strike.Back.S01E03.Mini.720p.HDTV.DD5.1.x264-ITSat.mkv
Strike.Back.S01E04.Mini.720p.HDTV.DD5.1.x264-ITSat.mkv
Strike.Back.S01E05.Mini.720p.HDTV.DD5.1.x264-ITSat.mkv
Strike.Back.S01E06.Mini.720p.HDTV.DD5.1.x264-ITSat.mkv

**Comment by vipuldelwadia** [comment]
Which of these shows were you expecting? http://www.thetvdb.com/?string=strike+back&searchseriesid=&tab=listseries&function=Search

When I tried it it matched with http://www.thetvdb.com/?tab=series&id=148581&lid=7

**Comment by djkrimson** [comment]
Sorry, I did not provide clear information. Details follow.

The Show is identified correctly, but the Episode title is incorrect. I checked the originals in thetvdb, they are correct. This happens in 0.7 as well as 0.7.1

e.g.

"Strike.Back.S01E01.Mini.720p.HDTV.DD5.1.x264.mkv" should be renamed as 
"Strike Back.S01E01.Chris Ryan's Strike Back, Episode 1.mkv"
not
"Strike Back.S01E01.Chris Ryan's Strike Back, Episode 2.mkv"

This sort of mismatch occurs for the rest of the files in the list in the first post as well.

![screen shot 2015-09-23 at 8 25 45 pm](https://cloud.githubusercontent.com/assets/14183528/10048778/329295b6-6232-11e5-8f58-087c5b9a1664.png)

![screen shot 2015-09-23 at 8 38 40 pm](https://cloud.githubusercontent.com/assets/14183528/10048979/2b2e67ea-6233-11e5-8990-f72f9211196a.png)
